GlyssenEngineTests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("GlyssenEngineTests")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("GlyssenEngineTests")]
+[assembly: AssemblyCopyright("Copyright ©  2019")]
```

Comment:
[assembly: AssemblyCopyright("Copyright © 2019 Faith Comes By Hearing and SIL International")]